testsuite/tests/input/tex/Bussproofs.test.ts

```diff
@@ -3209,3 +3209,5 @@ describe('BussproofsRegProofs', () => {
 </math>`
     ));
 });
+
+afterAll(() => getTokens('bussproofs'));
```

Comment:
I think you need to import `afterAll` and `getTokens` for this one.

Reply:
Done.